modules/ob-installing-builds-with-operator.adoc

```diff
@@ -0,0 +1,34 @@
+// This module is included in the following assembly:
+//
+// builds/installing-openshift-builds.adoc
+
+:_content-type: PROCEDURE
+[id="installing-builds-with-operator_{context}"]
+= Installing {builds-shortname} by using the Operator
+
+You can install the {builds-title} Operator from the OperatorHub by using the {ocp-product-title} web console. Installing this Operator enables you to install and use build components.
+
+.Procedure
+. In the *Administrator* perspective of the web console, navigate to the *Operators* -> *OperatorHub* page.
+
+. Use the *Filter by keyword* box to search for {builds-title} Operator in the catalog. 
+
+. Click the *{builds-title} Operator* tile.
+
+. Read the brief description about the Operator and click *Install*.
+
+. On the *Install Operator* page:
+
+.. The *Installation Mode* is set to *All namespaces on the cluster (default)*. This mode installs the Operator in the default `openshift-operators` namespace to watch and be made available to all namespaces in the cluster.
+
+.. The *Installed Namespace* is set to `openshift-operators` by default.
+
+.. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.
+
+.. The *Update Channel* is set to *v0.11.0* by default.
```

Comment:
```suggestion
.. The *Update Channel* is set to *latest* by default.
```
How about adding the details about Update Channel similar to what we have for OpenShift Pipelines in [Procedure](https://docs.openshift.com/container-platform/4.13/cicd/pipelines/installing-pipelines.html#op-installing-pipelines-operator-in-web-console_installing-pipelines) (section 4.c)?